### ISSUE TITLE
PW-Jack package within Pipewire

### ISSUE THREAD

**Issue by JamesFrederik** [clearlinux/distribution#2629]
Official package name:

PW-JACK / Pipewire

License (must be an OSI approved Open Source license):

GPL

Download URL of latest release:

https://gitlab.freedesktop.org/pipewire/pipewire

Latest release date (must be recent):

May 2022

Description:

Enable the PW-JACK module in Pipewire.

**Comment by JamesFrederik** [comment]
cheers, 

I actually managed to get 0.5.1 to compile with PW-JACK on my system today,  I think maybe the final GCC release fixed one error, and disabling the video system another. 

So I installed it I think into /usr/ which I suppose is quite bad.

But with minimal testing it's been working great, multiple JACK clients, including browser MIDI synths in Firefox all recording into Ardour, so far very stable and really quick. 

gonna be nice as pipewire matures and JACK like video patching apps get built :+1:
